pkg/util/v1alpha2/katibclient/katib_client.go

```diff
@@ -20,19 +20,25 @@ import (
 	"io/ioutil"
 	"strings"
 
-	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes/scheme"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
+	trialsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/trial/v1alpha2"
+	"github.com/kubeflow/katib/pkg/controller/v1alpha2/consts"
 )
 
 type Client interface {
 	InjectClient(c client.Client)
 	GetExperimentList(namespace ...string) (*experimentsv1alpha2.ExperimentList, error)
 	CreateExperiment(experiment *experimentsv1alpha2.Experiment, namespace ...string) error
+	DeleteExperiment(experiment *experimentsv1alpha2.Experiment, namespace ...string) error
+	GetExperiment(name string, namespace ...string) (*experimentsv1alpha2.Experiment, error)
 	GetConfigMap(name string, namespace ...string) (map[string]string, error)
+	GetTrialList(name string, namespace ...string) (*trialsv1alpha2.TrialList, error)
```

Comment:
Can we add DeleteTrial in the future? I think there is a need to delete the trial in DB after the trial is deleted in Kubernetes.